src/mono/System.Private.CoreLib/src/System/RuntimeType.Mono.cs

```diff
@@ -1327,6 +1327,8 @@ protected override bool IsValueTypeImpl()
         // Returns true for generic parameters with the Enum constraint.
         public override bool IsEnum => GetBaseType() == EnumType;
 
+        public override bool IsCollectible => RuntimeTypeHandle.GetAssembly(this)!.IsCollectible;
```

Comment:
Should this just return false given that collectible assemblies are not implemented in Mono?

This is not a correct implementation in case collectible assemblies were ever implemented in Mono. Correct implementation would have to take into account all generic instantiation arguments.